### PR TITLE
1863/feat/admin message

### PR DESCRIPTION
**Pull request by chenFriedman**

### CODE REVIEW

**Review by barnahum8 — APPROVED** [review]

**Review by barnahum8 — DISMISSED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ import InvestigationTableFooter from './InvestigationTableFooter/InvestigationTa
 import InvestigatorAllocationDialog from './InvestigatorAllocation/InvestigatorAllocationDialog';
 import useInvestigationTable, { SelectedRow, DEFAULT_SELECTED_ROW } from './useInvestigationTable';
 import { TableHeadersNames, TableHeaders, adminCols, userCols, Order, sortableCols, IndexedInvestigation } from './InvestigationTablesHeaders';
+import AdminMessages from './adminMessages/adminMessages';
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:33]: move import up

File: client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/adminMessages.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import { AdminMessage } from 'models/AdminMessage';
+import useAdminMessagesDBAction from './useAdminMessagesDBAction';
+import Desk from 'models/Desk';
+
+const AdminMessages: React.FC<Props> = (props: Props): JSX.Element => {
+  const classes = useStyles();
+  const { getAdminsMessagesByAdmin, adminsMessagesByAdmin, sendMessage, deleteMessage, toRefresh } = useAdminMessagesDBAction();
+
+  const messageTitle = 'הודעה';
+  const messageDescriptionTitle = '(תוצג בטבלת הדסק המסומן בעמוד זה)';
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/adminMessages.tsx:17]: Is it the right text?

File: client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/adminMessages.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                />
+              ))}
+            </div>
+          </Collapse> 
+        </Grid>
+    )
+}
+
+interface Props {
+  investigationInfoFilter: any
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/adminMessages.tsx:81]: why any

File: client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/useAdminMessagesDBAction.ts [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            id: 555,
+            admin_id: adminId,
+            desks_id: desksId
+        }
+        axios.post('/landingPage/sendMessage',
+            parameters).then((result) => {
+            setToRefresh(!toRefresh);
+            adminMessageLogger.info('new successfully', Severity.LOW);
+        }).catch((error) => {
+            alertError('לא הצלחנו לעדכן את ההודעה שלך');
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/useAdminMessagesDBAction.ts:57]: maybe move to const

File: client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/useAdminMessagesDBAction.ts [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    const deleteMessage = (id: number) => {
+        setIsLoading(true);
+        const adminMessageLogger = logger.setup('delete admin messages');
+        adminMessageLogger.info('launching DB request', Severity.LOW);
+        axios.post('/landingPage/deleteMessage',
+            id).then((result) => {
+            setToRefresh(!toRefresh);
+            adminMessageLogger.info('delete messeage successfully', Severity.LOW);
+        }).catch((error) => {
+            alertError('לא הצלחנו למחוק את ההודעה שלך');
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/useAdminMessagesDBAction.ts:73]: again move to const

File: client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/message/message.tsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    return (
+        <Grid container spacing={2}>
+          <Grid item xs={9}>
+            <TextField 
+              fullWidth
+              value={message}
+              onChange={event => setMessage(event.target.value as string)}
+              id="outlined-basic" 
+              variant="outlined"
+              placeholder="הקלד הודעה חדשה"
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/message/message.tsx:23]: replace " to '

**Review by Noam6421 — DISMISSED** [review]

File: client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/adminMessages.tsx (hunk 2 above)
Comment (chenFriedman) [reply client/src/components/App/Content/LandingPage/adminLandingPage/adminActions/adminMessages/adminMessages.tsx:17]: https://dev.azure.com/spectrumFactory/c29f56f9-1895-4190-94f9-3d13819ee54a/_apis/wit/attachments/452399c3-588f-43e8-951c-20b3d341e0f2?fileName=image.png
yes...

**Review by Noam6421 — APPROVED** [review]

**Review by barnahum8 — APPROVED** [review]

**Pull request merged**